packages/order-utils/src/index.ts

```diff
@@ -72,3 +72,5 @@ export {
     FeeOrdersAndRemainingFeeAmount,
     OrdersAndRemainingFillAmount,
 } from './types';
+
+export * from './rich_reverts';
```

Comment:
I'd rather we explicitly list the items we want exported. Someone might add an export to that file and be surprise to find that it's been auto-magically exported by the package.